src/Microsoft.DotNet.XHarness.CLI/Commands/WASM/Browser/WasmTestBrowserCommand.cs

```diff
@@ -265,7 +265,7 @@ protected override async Task<ExitCode> InvokeInternal(ILogger logger)
         if (Arguments.NoQuit)
             options.LeaveBrowserRunning = true;
 
-        logger.LogInformation($"Starting {driverName} with args: {string.Join(' ', options.Arguments)}");
+                logger.LogInformation($"Starting {driverName} with args: {string.Join(' ', options.Arguments)}");
```

Comment:
indentation looks wrong here, or did you mean to move this into the Arguments.NoQuit if?

Reply:
This should not have been touched, to be reverted.